.github/workflows/automatic-triage.yml

```diff
@@ -0,0 +1,13 @@
+name: Automatic triage
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: "0 12 * * 1" # Every Monday at 12:00 UTC
```

Comment:
Do we need to run this every week? What's the expected amount of new issues that will match this filter?